Description: Enable debugging in regression test shell script
Author: Jonas Smedegaard <dr@jones.dk>
Last-Update: 2011-05-25
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/test/rtpw_test.sh
+++ b/test/rtpw_test.sh
@@ -37,6 +37,9 @@
 # OF THE POSSIBILITY OF SUCH DAMAGE.
 #
 
+set -e
+set -x
+
 case $(uname -s) in
     *CYGWIN*|*MINGW*)
         EXE=".exe"
@@ -72,7 +75,7 @@
 # they are killed, those processes will linger.  Re-running the script
 # will get rid of them.
 
-pkill -x rtpw 2>/dev/null
+pkill -x rtpw 2>/dev/null || true
 
 if test -n $MESON_EXE_WRAPPER || test -x $RTPW; then
 
